projects/plugins/boost/app/lib/minify/functions-helpers.php

```diff
@@ -75,40 +73,22 @@ function jetpack_boost_init_filesystem() {
 
 function jetpack_boost_page_optimize_js_exclude_list() {
 	$exclude_list = jetpack_boost_ds_get( 'minify_js_excludes' );
-	if ( false === $exclude_list ) {
-		// Use the default since the option is not set
-		return jetpack_boost_page_optimize_js_exclude_list_default();
-	}
 	if ( '' === $exclude_list ) {
 		return array();
 	}
 
 	return explode( ',', $exclude_list );
 }
 
-function jetpack_boost_page_optimize_js_exclude_list_default() {
-	// WordPress core stuff, a lot of other plugins depend on it.
-	return Minify_JS::$default_excludes;
-}
-
 function jetpack_boost_page_optimize_css_exclude_list() {
 	$exclude_list = jetpack_boost_ds_get( 'minify_css_excludes' );
-	if ( false === $exclude_list ) {
-		// Use the default since the option is not set
-		return jetpack_boost_page_optimize_css_exclude_list_default();
-	}
 	if ( '' === $exclude_list ) {
 		return array();
 	}
 
 	return explode( ',', $exclude_list );
```

Comment:
If this is being exploded again, I wonder if an array is the true state of the excludes and it should only be shown as a string in the input field and then immediately exploded in JS?

Reply:
Yeah, makes sense to hold an array. I've updated it in a13eaf806dd706efa170bc8b8da768e7586dfb44.